android/aar/src/main/java/org/idpass/lite/android/IDPassLite.java

```diff
@@ -19,8 +20,16 @@
 
     private static final String shapeData = "shape_predictor_5_face_landmarks.dat";
     private static final String faceData = "dlib_face_recognition_resnet_model_v1.dat";
+    private static final String SHAPEPREDICTIONDATA = "SHAPEPREDICTIONDATA";
+    private static final String FACERECOGNITIONDATA = "FACERECOGNITIONDATA";
+    private static boolean isInitialized = false;
 
-    public static boolean loadModels(File cachedir, AssetManager am) {
+    public static boolean initialize(File cachedir, AssetManager am)
```

Comment:
@typelogic may I ask why there are two different classes with same name for IDPassLite?

Reply:
I intentionally made it same name. Since, Android has `IDPassLite::initialize` , then java has same method of same purpose